src/Compiler/Service/FSharpWorkspaceState.fs

```diff
@@ -221,6 +222,11 @@ module internal WorkspaceDependencyGraphExtensions =
             |> Seq.map this.GetValue
             |> _.UnpackMany(WorkspaceNode.projectSnapshot)
 
+        [<Extension>]
+        static member GetSourceFile(this: IDependencyGraph<_, _>, file) =
+            this.GetValue(WorkspaceNodeKey.SourceFile file)
+            |> _.Unpack(WorkspaceNode.sourceFile)
```

Comment:
Is "|> _." a modern way for writing a dot? Am I being punished for finishing the dot lambda? :-D

Reply:
It's the latter

Reply:
It's a _classy_ way to write a dot 🧐